sema/commons/yml/loader.py

```
@@ -0,0 +1,42 @@
+
+    def to_resolve(self, context):
+
+        def resolver(loader: SafeLoader, node: ScalarNode) -> str:
+            txt = loader.construct_scalar(node)
+            try:
+                txt = txt.format(**context)
+            except KeyError as ke:
+                log.error(
+                    f"yml config contains '{txt}' "
+                    f"with unknown key reference to --> {ke}"
+                )
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)